test/integration/logbucket-project/logbucket_project_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logbucket_project
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	defaultRetentionDays int64 = 30
+)
+
+func TestLogBucketProjectModule(t *testing.T) {
+
+	bpt := tft.NewTFBlueprintTest(t,
+		tft.WithTFDir("../../../examples/logbucket/project"),
+	)
+	bpt.DefineVerify(func(assert *assert.Assertions) {
+		bpt.DefaultVerify(assert)
+
+		for _, tc := range []struct {
+			projId             string
+			bktName            string
+			sinkDest           string
+			sinkProjId         string
+			sinkName           string
+			sinkWriterIdentity string
+		}{
+			{
+				projId:             bpt.GetStringOutput("log_bucket_project"),
+				bktName:            bpt.GetStringOutput("log_bucket_name"),
+				sinkDest:           bpt.GetStringOutput("log_sink_destination_uri"),
+				sinkProjId:         bpt.GetStringOutput("log_sink_project_id"),
+				sinkName:           bpt.GetStringOutput("log_sink_resource_name"),
+				sinkWriterIdentity: bpt.GetStringOutput("log_sink_writer_identity"),
+			},
+			{
+				projId:             bpt.GetStringOutput("log_bkt_same_proj"),
+				bktName:            bpt.GetStringOutput("log_bkt_name_same_proj"),
+				sinkDest:           bpt.GetStringOutput("log_sink_dest_uri_same_proj"),
+				sinkProjId:         bpt.GetStringOutput("log_sink_id_same_proj"),
+				sinkName:           bpt.GetStringOutput("log_sink_resource_name_same_proj"),
+				sinkWriterIdentity: "",
```

Comment:
```suggestion
```
default value for a variable of type string is ""

Reply:
I removed and left a comment explaning that in this example writerIdentity will be empty